src/RoslynInsertionTool/RoslynInsertionTool.Commandline/App.config

```diff
@@ -94,7 +94,7 @@
         <value>False</value>
       </setting>
       <setting name="UpdateXamlRoslynVersion" serializeAs="String">
-        <value>True</value>
+        <value>False</value>
```

Comment:
Should we update also `src/RoslynInsertionTool/RoslynInsertionTool.Commandline/Settings.settings`?